llvm/lib/Target/AMDGPU/SILoadStoreOptimizer.cpp

```diff
@@ -60,6 +60,7 @@
 #include "SILoadStoreOptimizer.h"
 #include "AMDGPU.h"
 #include "GCNSubtarget.h"
+#include "SIDefines.h"
```

Comment:
clang-format is not happy with this.

Reply:
Fixed. I do not follow why, though. I thought we always do it before deeper directories.